package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "qbit-race",
-  "version": "2.0.0-alpha.11",
+  "version": "2.0.0-alpha.13",
```

Comment:
```suggestion
  "version": "2.0.0-alpha.11",
```

Reply:
Actually, I see it is an inconsistency between the existing package.json and package-lock. Let me fix it in a separate MR so this diff goes away.